setup/react/app/ApplicationBootstrap.js

```diff
@@ -17,13 +17,15 @@ const ApplicationBootstrap = ({ children }) => {
   } = useGetDefaultApplication();
   const { setApplications } = useApplicationManagement();
   const [, setCurrentApplication] = useCurrentApplication();
+  const [isFirstTimeLoading, setIsFirstTimeLoading] = useState(true);
```

Comment:
why do we need this one?

Reply:
so what happens is, when we change application the UI flickers because isLoading would now be set to true by the `useGetDefaultApplication` hook. So I did this so we can always track first time boastrap of the application.